wallet/storage/src/lib.rs

```diff
@@ -34,18 +37,32 @@ pub type Error = storage::Error;
 pub trait WalletStorageRead {
     /// Get storage version
     fn get_storage_version(&self) -> Result<u32>;
-    fn get_utxo(&self, outpoint: &OutPoint) -> Result<Option<Utxo>>;
-    fn get_transaction(&self, id: &Id<Transaction>) -> Result<Option<WalletTx>>;
+    fn get_utxo(&self, outpoint: &AccountOutPointId) -> Result<Option<Utxo>>;
+    fn read_utxo_set(&self, account_id: &AccountId) -> Result<BTreeMap<AccountOutPointId, Utxo>>;
```

Comment:
Why do we have things call "read" and others called "get"?

Reply:
This is a relic from the chainstate storage. I went ahead and renamed those.